src/index.ts

```diff
@@ -144,7 +131,7 @@ function Pretender(/* routeMap1, routeMap2, ..., options*/) {
   self.XMLHttpRequest = interceptor(ctx);
 
   // polyfill fetch when xhr is ready
-  this._fetchProps = FakeFetch ? ['fetch', 'Headers', 'Request', 'Response'] : [];
```

Comment:
This was just added in a recently merged PR #235, so I'm surprised to see you changing it back. Have you studied the other PR, and verified that your change is correct?

Reply:
My initial impression was that it was just a fallback put in place to avoid an error state. (had seem to be a reaction to the issue linked to in the original PR)

Now I see that I was wrong, it is about restoring the global space to its original state **if** needed. I would have used different syntax, but that is out of scope for this PR :)

Reply:
The thing is, I still don't see how would `FakeFetch` ever be undefined? Originally, the conversation around it started because of issue #221, but that one is easily explained. The html wasn't loading `fetch.umd.js`.

I don't think we need the fallback, but I'm going to leave that in place since it isn't immediately pertinent to the objective of this PR.

Reply:
@xg-wang can you comment here as to the correct behavior? I don't want anyone to roll back a legitimate bugfix

Reply:
When `appearsBrowserified` is `false`, if the `whatwg-fetch` lib is not installed `self.WHATWGFetch` will be `undefined`. This always happens for non `ember-cli-pretender` users since they have to manual bundle the fetch lib.

In this pr the build output will be:
```js
var FakeFetch = self.WHATWGFetch;
var Pretender = (function (self, RouteRecognizer, FakeXMLHttpRequest, FakeFetch) {
}(self, RouteRecognizer, FakeXMLHttpRequest, FakeFetch));
```

The `self.WHATWGFetch` is pulled in by https://github.com/pretenderjs/pretender/blob/v2.1.1/karma.conf.js#L24